### DISCUSSION TITLE
Save the state of a view in Projects (beta)

### DISCUSSION THREAD

**Discussion by pratt33k** [community/community#13844] [Projects and Issues]
Problem example: If I remove the reviewers column from the table view and then come back to the original project. The review column comes back. I want the reviewers column to remain deleted. Essentially, save everything or atleast give an option to save the state

As a user
When I group by Sprint (iteration) or remove a column (like reviewers) or change anything else
I want to save the state of the view I am in
So that I don't have to change them again everytime I open the project.

**Comment by koanh01** [comment]


**Reply by pratt33k** [reply]
What do you 'do not know'?

**Comment by toandd7189** [comment]


**Reply by pratt33k** [reply]
Test what?

**Comment by garymm** [comment]
I think I'm seeing a bug where it's possible to save changes only when a view has a board layout but not on a table layout.
The blue dot that indicates changes are available to be saved is there but there's no "save changes" button.
